### PR TITLE
IBX-4759: Twig function to fetch current User

### PR DESCRIPTION
**Pull request by DominikaK**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | https://issues.ibexa.co/browse/IBX-4759
| Versions      | master (4.4)

#### Checklist

- [x] Text renders correctly
- [x] Text has been checked with vale
- [x] Code samples are working
- [x] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

File: docs/templating/twig_function_reference/other_twig_filters.md [hunk 1]
```diff
@@ -1,5 +1,20 @@
 # Other Twig filters
 
+### `ibexa_user_get_current()`
+
+`ibexa_user_get_current()` returns the User object of the current user.
```
Comment (Steveb-p) [inline docs/templating/twig_function_reference/other_twig_filters.md:5]: Specifically, what type of `User` object is returned? What is it's fully qualified class name?

Comment (DominikaK) [inline docs/templating/twig_function_reference/other_twig_filters.md:5]:
```suggestion
`ibexa_user_get_current()` returns the User object (`Ibexa\Core\Repository\Values\User\User`) of the current user.
```

File: docs/templating/twig_function_reference/other_twig_filters.md [hunk 2]
```diff
@@ -1,5 +1,20 @@
...
 
+### `ibexa_user_get_current()`
+
+`ibexa_user_get_current()` returns the User object (`Ibexa\Core\Repository\Values\User\User`) of the current user.
+
+``` html+twig
+{{ ibexa_user_get_current().login }}
+```
+
+You can get the underlying Content item, including among other the user's name,
```
Comment (MagdalenaZuba) [inline docs/templating/twig_function_reference/other_twig_filters.md:11]:
```suggestion
You can get the underlying Content item, for example the user's name,
```

**Review by MagdalenaZuba — APPROVED** [review]

File: docs/templating/twig_function_reference/other_twig_filters.md [hunk 3]
```diff
@@ -1,5 +1,20 @@
 # Other Twig filters
 
+### `ibexa_user_get_current()`
+
+`ibexa_user_get_current()` returns the User object (`Ibexa\Core\Repository\Values\User\User`) of the current user.
```
Comment (adamwojs) [inline docs/templating/twig_function_reference/other_twig_filters.md:5]:
```suggestion
`ibexa_user_get_current()` returns the User object (`Ibexa\Contracts\Core\Repository\Values\User\User`) of the current user.
```

**Review by adamwojs — CHANGES_REQUESTED** [review]

**Review by adamwojs — APPROVED** [review]

**Pull request merged**